evm/client/errors.go

```diff
@@ -296,7 +296,8 @@ var sei = ClientErrors{
 }
 
 var monad = ClientErrors{
-	Fatal: regexp.MustCompile("Gas limit too low"),
+	Fatal:          regexp.MustCompile("Gas limit too low"),
+	TooManyResults: regexp.MustCompile("Internal server error. Forwarder error: 1000."),
```

Comment:
Not sure that we should add an internal server error

Reply:
I don't see this as a problem.

It's unfortunate that this one is worded in such a general way. It could potentially refer to a more general class of errors than the one we're looking for.  But in this case, handling it as `TooManyResults` is erring on the safe side... by detecting it as that we're interpreting it as a temporary error that can be retried. Worst case, it keeps reducing the block range until it gets to 1 block and then gives up. So the only downside is if there is a permanent problem with the rpc server, it might take slightly longer to throw a critical error. The flipside is much worse, where reducing the block range would have helped but we gave up too easily without retrying.